src/components/LoadingMessage/LoadingMessage.less

```diff
@@ -0,0 +1,33 @@
+.lucid-LoadingMessage {
+
+	.lucid-Icon {
+		display: inline-block;
+	}
+
+	@LoadingMessage-padding: @size-standard;
```

Comment:
For other components we put the component-specific variables in the global scope rather than the component scope. do we want to break out of that pattern here?